modules/cnv-updating-access-mode-to-enable-live-migration.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * cnv/live_migration/cnv-live-migration.adoc
+
+[id="cnv-updating-access-mode-to-enable-live-migration_{context}"]
+= Updating access mode to enable LiveMigration
+
+Storage class defaults in {VirtProductName} are configured to provide ReadWriteOnce (RWO) access mode
+by default. Therefore, LiveMigration is not enabled by default unless you
+alter the access mode of your virtual machine.
+
+.Prerequisites
+* In order to use LiveMigration, virtual machines must have a persistent volume claim (PVC) with a shared
+ReadWriteMany (RWX) access mode to be live migrated.
+
+.Procedure
+To override the default RWO access mode, update the storage class configuration
+by doing the following:
+
+. Export the storage class object.
++
+----
+$ export STORAGE_CLASS="nfs"
+----
++
+. Update the storage class `kubevirt-storage-class-defaults` from RWO to RWX access mode.
++
+----
+$ oc patch -n openshift-cnv cm kubevirt-storage-class-defaults -p '{"data":{"'$<STORAGE_CLASS>'.accessMode":"ReadWriteMany"}}'
```

Comment:
The previous step explicitly exported the 'STORAGE_CLASS' variable. You don't need to wrap it in '<' '>' because it's now a literal variable rather than a replaceable one. (this is more of a FYI since I think we should drop the previous command anyway)

Reply:
Noted. Makes sense now.